src/avram/charms/json_extensions.cr

```diff
@@ -16,13 +16,21 @@ module JSON::Serializable
       value
     end
 
+    def parse(value : Array(JS)) forall JS
+      SuccessfulCast(Array(JS)).new value
```

Comment:
Actually, looks like I can use `T` here... I tried earlier and got a Lexer exception, but just tried again locally and it worked.. :thinking: